rfcs/000-resource-notices.md

```diff
@@ -0,0 +1,94 @@
+- Feature Name: Resource Notices
+- Start Date: 2021-03-22
+- RFC PR: [amundsen-io/rfcs#29](https://github.com/amundsen-io/rfcs/pull/29) (after opening the RFC PR, update this with a link to it and update the file name)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Resource Notices
+
+## Summary
+
+Resource Notices allows Amundsen administrators to show informational, warning, and alert messages related to the different resources (tables, dashboards, people) we expose in Amundsen.
+
+## Motivation
+
+This feature will help administrators show messages related to deprecation, updates (or lack of), and informational messages related to specific resources. In Lyft's case, we use them to point final users to improved versions of current surfaced tables, so final users employ more trusted data.
+
+## Guide-level Explanation (aka Product Details)
+
+An Amundsen final user can see a notice in the table/dashboard/user detail page whenever their system admin adds one related to it.
+
+A notice is a small box with an icon and a message containing HTML markup (like links and bolded text).
+
+These will come in three flavors:
+
+- Informational: Marked with a blue "i" icon on the right side
+- Warning: Marked with an orange exclamation mark icon on the right side
+- Alert: Marked with a red exclamation mark icon on the right side
+
+For example, if company X wants to deprecate the use of one table, they can opt to add a new notice in their configuration file:
+
+```
+    ... Table Configuration
+    notices: {
+        "<SCHEMANAME>.<TABLENAME>": {
+          severity: NoticeSeverity.ALERT,
+          message: `<span>This table is deprecated, please use <a href="<LINKTONEWTABLEDETAILPAGE>">this new table</a> instead.</span>`,
+        },
+    },
+```
+
+This code will show a notice with a red exclamation icon whenever a final user visits the table's Table Detail page.
+
+This feature's ultimate goal is to allow Amundsen administrators to point their users to more trusted/higher quality data without removing the old references.
```

Comment:
higher quality would be just one use-case I think, this can easily be used for
* recommending different tables (`more like this` feature or `other users also queried`)
* notifying about failed data quality checks
* notifying about maintenance on the tables (like it's being repartitioned and it will take X hours)

Reply:
Those are great examples, thanks!